### PR TITLE
support for other 32bit arm arch

### LINKED ISSUE #21: Does not work with 32bit arm
It looks like there is support for 64bit arm (arch aarch64) but not 32bit arm (arm, arm7, etc) I think it would be as simple as adding a clause for "arm".

I tried this out and it appears to work: https://github.com/phoenixframework/esbuild/compare/main...hez:feature/linux-arm?expand=1#diff-817be7445555ff97e4b7a5c8139e8e7e953cd62b9175b157793e979ad85dd79dR193

### PR DESCRIPTION
**Pull request by hez**
fixes #21

### CODE REVIEW

**Comment by josevalim** [comment]
Thanks. I will merge it and do a new release tomorrow!

File: lib/esbuild.ex [hunk 1]
```diff
@@ -187,7 +187,6 @@ defmodule Esbuild do
           "amd64" -> "#{osname}-64"
           "x86_64" -> "#{osname}-64"
           "aarch64" -> "#{osname}-arm64"
-
```
Comment (hez) [inline lib/esbuild.ex]: oh! drat good catch!

**Comment by hez** [comment]
Thank you for the quick turn around!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**